vnets.tf

```diff
@@ -166,3 +166,40 @@ resource "azurerm_virtual_network_peering" "private_public" {
   allow_gateway_transit        = false
   use_remote_gateways          = false
 }
+
+##################################################################################################################
+###################   TRUSTED   ##################################################################################
+##################################################################################################################
+
+# Resources groups for TRUSTED NETWORK
+resource "azurerm_resource_group" "trusted" {
+  name     = "jenkinsinfra-trusted"
+  location = "East US"
+}
+
+# NETWORKING FOR TRUSTED
+resource "azurerm_virtual_network" "trusted" {
+  name                = "trusted-controller-network"
+  address_space       = ["10.246.0.0/16"]
```

Comment:
Overlapping with `public` vnet, need to be changed.

WDYT of including the dedicated subnets inside the `private` vnet instead?

Reply:
If not, you'll have to create a vnet above the last IP from `private` vnet (ie starting at 10.252.0.0)

Reply:
didn't realize the overlapping from 10.244.0.0/14 ... thanks to pointing out

Reply:
we decided together to create a separate vnet from public AND private